### PR TITLE
Install Theme no matter if the user run the npx create-wp-project command in the wrong folder

### PR DESCRIPTION
**Pull request by DjidjaleskaSandra**
# Description
The task aims to create a Node.js function that identifies the root directory of a WordPress project by searching for the presence of the wp-config.php file. This functionality will facilitate automated processes within the project setup or maintenance scripts, ensuring they execute from the correct directory.

# Screenshots / Videos
https://github.com/infinum/eightshift-frontend-libs/assets/97440314/7c1c7af7-cf2e-4f3e-82af-d0ec79a4c18b

# Productive
https://app.productive.io/1-infinum/tasks/7791999

### CODE REVIEW

File: setup/create-wp-project/src/commands/theme.js
Comment (iruzevic) [inline setup/create-wp-project/src/commands/theme.js:42]: The goal is to allow users to set up the project from the root directory, without being required to be inside the theme folder. The installation process will ensure that the theme is placed in the correct location.

So if they setup the project from the root, wp-content, wp-content/plugins no mater the setup should go to the root of the project where the WP-config is set add add the theme to the correct place.

The same thing should work with the plugin flag used for the plugin setup

**Review by iruzevic — CHANGES_REQUESTED** [review]

File: setup/create-wp-project/src/commands/theme.js
Comment (iruzevic) [inline setup/create-wp-project/src/commands/theme.js:41]: how will this work with the plugin flag?

**Review by iruzevic — CHANGES_REQUESTED** [review]

File: setup/create-wp-project/src/commands/theme.js
Comment (DjidjaleskaSandra) [reply setup/create-wp-project/src/commands/theme.js]: I think Plugins to be in a separate task, but putting everything together into one PR can also work, especially when the changes are closely connected and won't get confusing. So, we can go ahead with that approach :).

**Review by iruzevic — APPROVED** [review]
looking good, please wait for the review from @goranalkovic-infinum

**Comment by iruzevic** [comment]
I have used parts of your PR for the new release so I will not merge this but parts of this is in the new release. tnx

**Pull request closed without merging**
